src/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -235,5 +235,119 @@ public static class MsalError
         /// the configured redirect uri.
         /// </summary>
         public const string CustomWebUiRedirectUriMismatch = "custom_webui_invalid_mismatch";
+
+        /// <summary>
+        /// Access denied.
+        /// </summary>
+        public const string AccessDenied = "access_denied";
+
+        /// <summary>
+        /// JSON Parse error.
```

Comment:
The description is not good enough. We need to explain when this happens and what remediation there is, if any.